package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "tiny-lobster",
+  "version": "0.1.0",
+  "description": "My teensy-tinsy node.js pet",
+  "main": "index.js",
+  "repository": "git@github.com:octocrux/tiny-lobster.git",
+  "author": "octocrux <cate.petrusevich@gmail.com>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist/* && mkdir -p dist",
+    "start": "yarn build && PORT=3001 NODE_ENV=production node dist/app.js",
+    "build": "yarn clean && tsc --noEmit false --outDir dist",
+    "dev": "PORT=3000 NODE_ENV=dev nodemon --config nodemon.json src/app.ts",
```

This command is not cross-platform. Use crossenv.
```suggestion
    "dev": "crossenv PORT=3000 NODE_ENV=dev nodemon --config nodemon.json src/app.ts",
```